CHANGES.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- [IMPROVED] Faster `Result` iterator.
```

Comment:
We should probably give some detail and not use 'Faster'
i.e. `Improve Result iteration by paginating with 'startkey'`.  @smithsz wdyt?